src/dumper.rs

```diff
@@ -189,6 +189,11 @@ impl<'a> Dumper<'a> {
                 current_size = base.size();
                 (rem, Some(KnownBase::ISerializable))
             }
+            [base, rem @ ..] if base.name() == CName::new("gameScriptableSystem") => {
+                writeln!(out, "    pub base: ScriptableSystem,")?;
+                current_size = base.size();
+                (rem, Some(KnownBase::ScriptableSystem))
+            }
```

Comment:
```suggestion
            [base, rem @ ..] if base.name() == CName::new("gameScriptableSystem") => {
                writeln!(out, "    pub base: ScriptableSystem,")?;
                current_size = base.size();
                (rem, Some(KnownBase::ScriptableSystem))
            }
```
this code won't work because `gameScriptableSystem` is not a top-most type, what is the reason for adding this?

Reply:
the reason is that I was in a rush and largely glossed over the code, applying identical changes from `IScriptable` without having time to review everything thoroughly. thanks for pointing out, let me revert this change.

Reply:
reverted in f389346.